fdm-app/app/components/custom/combobox.tsx

```
@@ -68,85 +70,85 @@ export function Combobox({
-                        </PopoverTrigger>
-                        <PopoverContent
-                            id="combobox-options"
-                            className="w-full p-0"
+        <FormItem>
+            <FormLabel>{label}</FormLabel>
+            <Popover open={open} onOpenChange={setOpen}>
+                <PopoverTrigger asChild>
+                    <FormControl>
+                        <Button
+                            variant="outline"
+                            role="combobox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)